Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM python:2
 
 # Install necessary packages
 RUN pip install buildbot requests
+RUN pip install txrequests git+https://github.com/iblis17/buildbot-freebsd.git
```

Comment:
Does this need to be installed on the master?

Reply:
I have some custom `steps`, like
https://github.com/JuliaCI/julia-buildbot/pull/87/files#diff-43e4fe43d9ca1e0a77951b0b5fecfc40R22.
These steps is provided by that plugin.

Maybe `txrequests` isn't needed.

Reply:
http://docs.buildbot.net/latest/manual/cfg-reporters.html#githubstatuspush
GitHubStatusPush requires `txrequests`